### PR TITLE
Add xlsx export to views

### PR DESCRIPTION
**Pull request by svenow**
xlsx export functionality has been added to the views

- funding application list
- funding application process list

xlsx export functionality has been added via Drupal modules

- drupal/views_data_export:^1.4
- drupal/xls_serialization:^1.4

Drupal modules have been added as dependencies to this module

### CODE REVIEW

**Comment by svenow** [comment]
Refers to Systopia Issue #24751

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
-langcode: en
+langcode: de
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:1]: This should still be `en`.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 2]
```diff
@@ -1036,7 +1038,59 @@ display:
             fail: 'not found'
           validate_options: {  }
           must_not_be: false
-      filters: {  }
+      filters:
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:1041]: Is this related to the export?

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 3]
```diff
@@ -1,10 +1,14 @@
-langcode: en
+langcode: de
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1]: This should still be `en`.

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 4]
```diff
@@ -1048,57 +1052,10 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
-        application_process_status:
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1051]: I wouldn't expect this to be removed...

**Review by dontub — CHANGES_REQUESTED** [review]
I'm currently not sure if this PR makes also unrelated changes. In the views I'd actually expect not much more than new module dependencies and the tree `data_export_1`.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml (hunk 1 above)
Comment (svenow) [reply config/optional/views.view.civiremote_funding_combined_application_process_list.yml:1]: this probably comes from the drupal views export as the site is configured to german language. i will switch language in the exported view back to en

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml (hunk 2 above)
Comment (svenow) [reply config/optional/views.view.civiremote_funding_combined_application_process_list.yml:1041]: this status filter was configured already in the dev environment but i guess its not related to this feature. i can remove it but will add it again to another feature, where a status filter was requested in another view (based on the experience with the filter in this view)

File: config/optional/views.view.civiremote_funding_application_list.yml (hunk 3 above)
Comment (svenow) [reply config/optional/views.view.civiremote_funding_application_list.yml:1]: will switch back to en

File: config/optional/views.view.civiremote_funding_application_list.yml (hunk 4 above)
Comment (svenow) [reply config/optional/views.view.civiremote_funding_application_list.yml:1051]: i see, mixup of the removal of the status in combined application list and this view. i will revert the deletion here

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 5]
```diff
@@ -1129,15 +1090,15 @@ display:
               empty_column: false
               responsive: ''
             application_process_id:
-              sortable: false
+              sortable: true
```
Comment (svenow) [inline config/optional/views.view.civiremote_funding_application_list.yml:1093]: `sortable: true` may has been introduced by another view feature as well. shall it be reverted to `false`?

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 6]
```diff
@@ -1129,15 +1090,15 @@ display:
...
               default_sort_order: asc
               align: ''
               separator: ''
               empty_column: false
               responsive: ''
             funding_case_id:
               sortable: true
-              default_sort_order: desc
+              default_sort_order: asc
```
Comment (svenow) [inline config/optional/views.view.civiremote_funding_application_list.yml:1101]: the same with the sort order, probably introduced for another view feature

Comment (svenow) [reply config/optional/views.view.civiremote_funding_application_list.yml]: switched back to asc

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 7]
```diff
@@ -1146,6 +1204,80 @@ display:
...
+            amount_requested:
+              alias: ''
+              raw_output: true
+            amount_cleared:
+              alias: ''
+              raw_output: true
+            amount_admitted:
+              alias: ''
+              raw_output: true
+            dropbutton:
```
[the remark this answers is not in the document]
Comment (svenow) [reply config/optional/views.view.civiremote_funding_combined_application_process_list.yml:1258]: yes, in the field-export-configuration of that view the drop button is listed. i didnt see an option to exclude it from the list of exported fields. manual removal would be an option here but might be overwritten again on next export.

**Review by dontub — APPROVED** [review]
